std/socket.d

```diff
@@ -132,7 +132,7 @@ version(unittest)
     {
         try
             test();
-        catch (Throwable e)
+        catch (Exception e)
```

Comment:
The whole point of this is clearly to catch `Throwable` - including `AssertError`. So, this is definitely wrong.